SU2_CFD/src/output/CFlowIncOutput.cpp

```diff
@@ -318,6 +318,8 @@ void CFlowIncOutput::SetVolumeOutputFields(CConfig *config){
 
   if (config->GetKind_Solver() == MAIN_SOLVER::INC_RANS || config->GetKind_Solver() == MAIN_SOLVER::INC_NAVIER_STOKES){
     AddVolumeOutput("LAMINAR_VISCOSITY", "Laminar_Viscosity", "PRIMITIVE", "Laminar viscosity");
+    AddVolumeOutput("HEAT_CAPACITY", "Heat_Capacity", "PRIMITIVE", "Heat capacity");
+    AddVolumeOutput("EFFECTIVE_THERMAL_CONDUCTIVITY", "Effective_Thermal_Conductivity", "PRIMITIVE", "Effective thermal conductivity");
```

Comment:
For consistency with the custom output variables
```suggestion
    AddVolumeOutput("THERMAL_CONDUCTIVITY", "Effective_Thermal_Conductivity", "PRIMITIVE", "Effective thermal conductivity");
```